### ISSUE TITLE
Use composable JSON rather than building JSON through strings

### ISSUE THREAD

**Issue by gallimorej** [gallimorej/thankser#14]
https://github.com/gallimorej/thankser/commit/9fb384a61e2c0d9f94d8bbe9f949ce9f9f5c646b

**Comment by realgenekim** [comment]
This does indeed work.

```
(json/write-str {:response_type "in-channel"
                 :text "abc"})
=> "{\"response_type\":\"in-channel\",\"text\":\"abc\"}"
```

**Comment by gallimorej** [comment]
Fixed with commit: 399bf08ee3c2404e924c84656c8160ff2a046112
